sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -14,9 +14,9 @@
 // Last bits are not provided by the generated header.
 // TODO: https://github.com/lowRISC/opentitan/issues/3932
 #define CLKMGR_CLK_ENABLES_LAST_BIT CLKMGR_CLK_ENABLES_CLK_USB_PERI_EN_BIT
-#define CLKMGR_CLK_HINTS_LAST_BIT CLKMGR_CLK_HINTS_CLK_MAIN_OTBN_HINT_BIT
+#define CLKMGR_CLK_HINTS_LAST_BIT CLKMGR_CLK_HINTS_CLK_MAIN_KMAC_HINT_BIT
 #define CLKMGR_CLK_HINTS_STATUS_LAST_BIT \
-  CLKMGR_CLK_HINTS_STATUS_CLK_MAIN_OTBN_VAL_BIT
+  CLKMGR_CLK_HINTS_STATUS_CLK_MAIN_KMAC_VAL_BIT
```

Comment:
Note that changes to the clkmgr unit test are only needed due to moving OTBN's clock group. This is just to allow a successful CI run for now. We need to investigate the main issue (why I can't used the 'fixed' clock group) in more detail.